build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/plugin_list.rb

```diff
@@ -54,6 +59,7 @@ def default_plugins
         'Aws::Plugins::ResponsePaging' => "#{core_plugins}/response_paging.rb",
         'Aws::Plugins::StubResponses' => "#{core_plugins}/stub_responses.rb",
         'Aws::Plugins::IdempotencyToken' => "#{core_plugins}/idempotency_token.rb",
+        'Aws::Plugins::InvocationId' => "#{core_plugins}/invocation_id.rb",
```

Comment:
Are we sure this should be on regular clients?

Reply:
Yes